packages/code-du-travail-api/src/server/routes/themes/index.js

```diff
@@ -21,12 +35,15 @@ const router = new Router({ prefix: API_BASE_URL });
  *
  * @returns {Object} An object containing the matching theme .
  */
-router.get("/themes", ctx => {
-  ctx.body = {
-    label: "Thèmes",
-    children: getChildren(null)
-  };
+router.get("/themes", async ctx => {
+  const body = getRootThemesQuery({});
+  const response = await elasticsearchClient.search({
+    index: indexThemes,
+    body
+  });
+  ctx.body = response.body;
```

Comment:
I wonder if we should start hiding ES from now and  unwrap data here to avoid the `results.hits.hits`